packages/app/src/sandbox/compile.ts

```diff
@@ -439,9 +440,9 @@ async function compile({
 
   hadError = false;
 
-  actionsEnabled = hasActions;
+  actionsEnabled = !!hasActions;
```

Comment:
Using `Boolean` instead of double negation is a bit clearer I think

```suggestion
  actionsEnabled = Boolean(hasActions);
```